draft-ietf-tls-external-psk-guidance.md

```diff
@@ -425,8 +428,12 @@ unique identifier can, for example, be one of its MAC addresses, a 32-byte
 random number, or its Universally Unique IDentifier (UUID) {{?RFC4122}}. Each
 endpoint SHOULD know the identifier of the other endpoint with which its wants
 to connect and SHOULD compare it with the other endpoint’s identifier used in
-ImportedIdentity.context. It is however important to remember that endpoints
-sharing the same group PSK can always impersonate each other.
+ImportedIdentity.context. Note that the globally unique identifier is used only
```

Comment:
If the identifier is included in ImportedIdentity.context, then it is sent on the wire (as the imported PSK's identity), so this new text doesn't seem right.

Reply:
Oh, oops, good point.  I think my mental model for the importer was that the PSK identity sent on the wire was an HKDF output rather than something with structure.
I guess we just have to revert this chunk, then?

Reply:
Yeah, I think so. The rest seems fine to me!